chromadb/server/fastapi/__init__.py

```diff
@@ -70,6 +70,7 @@ def __init__(self, settings):
         self.router.add_api_route("/api/v1", self.root, methods=["GET"])
         self.router.add_api_route("/api/v1/reset", self.reset, methods=["POST"])
         self.router.add_api_route("/api/v1/version", self.version, methods=["GET"])
+        self.router.add_api_route("/api/v1/heartbeat", self.heartbeat, methods=["GET"])
```

Comment:
added an explicit url for this to make it easy for our OpenAPI spec to pick it up

Reply:
the python frontend is still using the "old" route... `self.router.add_api_route("/api/v1", self.root, methods=["GET"])`, but we could move it over to the new one as well. perhaps it makes sense to do that in this PR? Though having isolation b/t JS and python changes is nice.